src/components/customerPortal/PortalCustomerInfos.tsx

```diff
@@ -22,198 +32,209 @@ gql`
       country
       city
       zipcode
+
+      shippingAddress {
+        addressLine1
+        addressLine2
+        city
+        country
+        state
+        zipcode
+      }
     }
   }
 `
 
-interface PortalCustomerInfosProps {
-  translate: Function
+const FieldTitle = ({ title }: { title: string }) => (
+  <p className="text-sm font-normal leading-6 text-grey-600">{title}</p>
```

Comment:
Why not using the Typography component we have in our design system?
IIRC there is an attribut you could pass to make it use a `p` and not a `div` as it does by default

Maybe it raise the question of the usage of such typo component as we have those tailwind classes now, but I would rather make a large migration later than starting to introduce some different implementation WDYT?

Reply:
Good suggestion, but I would also keep this as a future improvement. Currently <Typography> brings lots of functionality behind the scenes and since all the typography in the portal was mapped to the Figma design, I'm afraid that this might include un-wanted changes.

Reply:
Awesome